src/core/components/info.jsx

```diff
@@ -82,13 +83,13 @@ export class InfoUrl extends React.PureComponent {
     getComponent: PropTypes.func.isRequired
   }
 
-  
+
   render() {
     const { url, getComponent } = this.props
 
     const Link = getComponent("Link")
 
-    return <Link target="_blank" href={ sanitizeUrl(url) }><span className="url"> { url } </span></Link>
+    return <Link target="_blank" href={ sanitizeUrl(url) }><span className="url"> { url } !!!</span></Link>
```

Comment:
**Question**: is this an intentional change?

Reply:
No apologies